packages/test-utils/src/vue-wrapper.js

```diff
@@ -11,17 +12,17 @@ export default class VueWrapper extends Wrapper implements BaseWrapper {
     // $FlowIgnore : issue with defineProperty
     Object.defineProperty(this, 'vnode', {
       get: () => vm._vnode,
-      set: () => {}
+      set: () => throwError(`VueWrapper.vnode is read-only`)
```

Comment:
This can be a Wrapper or a VueWrapper, I think just go with:

`wrapper.vnode is read-only`